sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/TextAnalyticsActionResult.java

```diff
@@ -12,7 +13,7 @@
 /**
  * The {@link TextAnalyticsActionResult} model.
  */
-public class TextAnalyticsActionResult {
+public abstract class TextAnalyticsActionResult {
```

Comment:
Can you add some usage samples and testcases? It will provide more context on how this change will impact the user experience.

Reply:
talked offline. We are agreed to remove TextAnalyticsActionResult  class. There is no need to maintain TextAnalyticsActionResult class since this class will not be used by the users directly. If this class can be used directly by users. then we can introduce it and it won't be a breaking change.